### DISCUSSION TITLE
Why are there two different packages for react tsparticles?

### DISCUSSION THREAD

**Discussion by IDrumsey** [matteobruni/tsparticles#4821] [General]
There seems to be two different packages for the react version of tsparticles.

1. [react-particles](https://www.npmjs.com/package/react-particles)
2. [react-tsparticles](https://www.npmjs.com/package/react-tsparticles)

Is react-particles just for using with javascript, because when I tried to use it, I got an error saying the type declarations weren't found. So I did some digging and came across the second one (react-tsparticles). The documentation seems to point to the react-particles package. See [here](https://github.com/matteobruni/tsparticles#reactjs) and [here](https://github.com/matteobruni/tsparticles/tree/main/components/react#installation). If you do a search for 'react-tsparticles' on either of those web pages, it doesn't seem to be mentioned. It is mentioned in the changelog up to version 2.1.4 (2022-07-28) - see [here](https://github.com/matteobruni/tsparticles/blob/main/components/react/CHANGELOG.md#214-2022-07-28).

A few questions - 
1. If I'm using Typescript in react, is react-tsparticles the correct package to be using?
2. The changelog suggests version bumps for react-tsparticles up to 2.1.3 - see [here](https://github.com/matteobruni/tsparticles/blob/main/components/react/CHANGELOG.md#214-2022-07-28:~:text=Note%3A-,Version%20bump%20only%20for%20package%20react%2Dtsparticles,-2.1.2%20(2022%2D07)), and then after that it seems that react-particles is the only package getting a version bump - see[here](https://github.com/matteobruni/tsparticles/blob/main/components/react/CHANGELOG.md#214-2022-07-28:~:text=Note%3A-,Version%20bump%20only%20for%20package%20react%2Dparticles,-2.2.0%20(2022%2D08)), but react-tsparticles is at the same version as react-particles on npm. Are they being linked automatically or something? This may be a lack of understanding of npm... idk.

Searching for similar discussions pointed me to [here](https://github.com/matteobruni/tsparticles/discussions/1125), but react-particles doesn't seem to be mentioned there.

**Comment by matteobruni** [comment]
They are the same package with a different name, I prefer using `react-particles` for the React package, the old `react-tsparticles` is still supported for now.

Both share the same codebase, I've made a sample using React + TS here: https://codesandbox.io/s/sweet-perlman-xzi5do?file=/src/App.tsx

As you can see it uses `react-particles` without issues.
